### PR TITLE
[go.mod] update kr/pretty to latest upstream

### PR DESCRIPTION
**Pull request by barrettj12**
Our patch for kr/pretty has now been merged upstream, so I've removed the
```
replace github.com/kr/pretty => github.com/canonical/pretty
```
in go.mod, and updated kr/pretty to the latest version (containing the patch).

### CODE REVIEW

**Review by benhoyt — APPROVED** [review]
Thanks!

**Comment by barrettj12** [comment]
Will wait till CI is fixed before merging as I want to get a full CI run (make sure nothing's broken)

**Pull request merged**
